packages/office-addin-project/package.json

```diff
@@ -19,9 +22,9 @@
     "commander": "^6.2.1",
     "fs-extra": "^7.0.1",
     "inquirer": "^7.3.3",
-    "office-addin-manifest": "^1.10.1",
-    "office-addin-manifest-converter": "^0.1.4",
-    "office-addin-usage-data": "^1.6.1",
+    "office-addin-manifest": "^1.10.0",
```

Comment:
Why are we down grading these versions?

Reply:
Are they released? They are not the latest and I just followed what VSCode recommended

Reply:
I didn't check what the specific current versions are, but they shouldn't be updated just because.  The version that has been set should remain unless there is a reason to change it.  Presumably if they were changed to that version there was a reason for it and we don't want to break that unless we have a specific reason to do so.